package.json

```diff
@@ -17,19 +18,24 @@
     "lint": "eslint --max-warnings=0 .",
     "lint-fix": "eslint --fix .",
     "test": "node test.js | tap-parser -l",
+    "type": "tsc",
     "bump": "./bump.sh"
   },
   "files": [
     "/x-element.js",
+    "/x-element.d.ts",
+    "/x-element.d.ts.map",
     "/demo",
     "/test",
     "/etc"
   ],
   "devDependencies": {
     "@netflix/eslint-config": "^3.0.0",
+    "eslint-plugin-jsdoc": "^50.3.1",
```

Comment:
After using this for a little while — I feel like this is a pretty good solution. Open to other suggestions though. FWIW, I tried using TypeScript to generate errors here and I kept feeling like it wasn’t doing what I wanted it to do — could be that I had it set up wrong though!